website/content/en/docs/scorecard/scorecard-alpha.md

```diff
@@ -0,0 +1,221 @@
+
+## Requirements
+
+The scorecard tests make no assumptions as to the state of the 
+operator being tested.  Creating operators and custom resources 
+for an operator are left outside the scope of the scorecard itself.
+
+Scorecard tests can however create whatever resources they 
+require if the tests are designed for resource creation.
+
+## Running the Scorecard
+
+1. Define a scorecard configuration file `config.yaml`.  A sample 
+configuration file [config.yaml][sample-config] is found within 
+the SDK repository. See [Config file](#config-file) for an explaination 
+of the configuration file format.  Unless you are executing custom 
+tests, you can just copy the provided example configuration file 
+into your project.
+2. Place the scorecard configuration file within your project 
+bundle directory a the following location `tests/scorecard/config.yaml`.  
+You can override the default location of the configuration file 
+by specifying the `--config` flag.
+3. Execute the [`scorecard` command][cli-scorecard]. See the 
+[Command args](#command-args) to check its options.
+
+## Configuration
+
+The scorecard test execution is driven by a configuration file, 
+`config.yaml`.   The configuration file is located at the following 
+location within your bundle:
+```
+tests/scorecard/config.yaml
+```
+
+### Config File
+
+A complete scorecard configuration file can be found [here][sample-config] 
+and used for running the scorecard pre-defined tests that ship with the SDK.
+
+A sample of the scorecard configuration file may look as follows:
+
+```yaml
+tests:
+- name: "basic-check-spec"
+  image: quay.io/operator-framework/scorecard-test:dev
+  entrypoint:
+  - scorecard-test
+  - basic-check-spec
+  labels:
+    suite: basic
+    test: basic-check-spec-test
+  description: check the spec test
+- name: "olm-bundle-validation"
+  image: quay.io/operator-framework/scorecard-test:dev
+  entrypoint:
+  - scorecard-test
+  - olm-bundle-validation
+  labels:
+    suite: olm
+    test: olm-bundle-validation-test
+  description: validate the bundle test
+```
+
+The configuration file defines each test that scorecard can execute.  The 
+following fields of the scorecard configuration file define the test 
+as follows:
+
+ 
+| Config Field        | Description   |
+| --------    | -------- |
+| image | the test container image name that implements a test 
+| entrypoint | the command and arguments that are invoked in the test image to execute a test
+| labels | user defined labels that allow for test selection using the scorecard CLI `--selector` flag
+
+### Command Args
+
+The scorecard command has the following syntax:
+```
+operator-sdk alpha scorecard [bundle path] | [bundle image name] [flags]
+```
+
+The scorecard requires a positional argument that holds either the
+on-disk path to your operator bundle or the name of a bundle image.
+
+**NOTE:** For further information about the flags see the [CLI documentation][cli-scorecard].
```

Comment:
Using a bolded "NOTE" is unnecessary for something that flows from previous context:
```suggestion
For further information about the flags see the [CLI documentation][cli-scorecard].
```